btc-transaction-helper.js

```diff
@@ -272,6 +272,15 @@ class BtcTransactionHelper {
         return Buffer.concat([bufferVersion, decodedAddress.hash]).toString('hex');
     };
 
+    /**
+     *
+     * @param {string} rawTransaction string in hex to be parsed into an object
+     * @param {boolean} isWitness string in hex to be parsed into an object
```

Comment:
?

Reply:
`isWitness` is an optional parameter that can be passed for decoding a raw transaction to the btc node client, but then I realized is not needed. See more info on how it works here: https://chainquery.com/bitcoin-cli/decoderawtransaction

Reply:
Unknown parameter description removed